smqtk_relevancy/impls/relevancy_index/classifier_wrapper.py

```diff
@@ -36,31 +39,32 @@ class SupervisedClassifierRelevancyIndex (RelevancyIndex):
         clone at rank time. The input classifier instance is not modified.
     """
 
-    def __init__(self, classifier_inst):
+    def __init__(self, classifier_inst: SupervisedClassifier):
         super(SupervisedClassifierRelevancyIndex, self).__init__()
         self._classifier_type = type(classifier_inst)
         self._classifier_config = classifier_inst.get_config()
         # Some number of descriptors to be ranked, cached as the elements
         # themselves as well as a vertically-stacked matrix (ndim==2).
         # These are None when there is no index yet.
-        self._descr_elem_list = None
-        self._descr_matrix = None
+        self._descr_elem_list = None  # type: Optional[List[DescriptorElement]]
```

Comment:
Not a thing to necessarilly be "corrected" here, but FYI you can annotate variables like
```suggestion
        self._descr_elem_list: Optional[List[DescriptorElement]] = None
```

Reply:
went ahead and resolved this bc it looks cleaner